server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -137,7 +188,13 @@ private void registerAsyncMetrics(MeterRegistry registry) {
                 "es.translog.uncommitted_operations.total",
                 "Number of uncommitted transaction log operations.",
                 "operations",
-                () -> new LongWithAttributes(stats.getOrRefresh().getIndices().getTranslog().getUncommittedOperations())
+                () -> new LongWithAttributes(
+                    Optional.ofNullable(stats.getOrRefresh())
+                        .map(o -> o.getIndices())
+                        .map(o -> o.getTranslog())
+                        .map(o -> o.getUncommittedOperations())
+                        .orElse(0)
```

Comment:
NIT: `orElse(0L)`

Reply:
same as above: `getUncommittedOperations()` returns and `int`, this means that the `.map` returns and Optional<Integer>.
Returning a `long` would not compile.